### PR TITLE
Update protos and client's configuration methods with stable config api

### PR DESCRIPTION
**Pull request by hunter007**
the PR should be merged after #389

### CODE REVIEW

**Comment by shivamkm07** [comment]
subscribe method for config api needs to be updated as per sdk-spec(https://github.com/dapr/sig-sdk-spec/blob/main/spec/alpha/configuration.md). I have created PR for the same: #389 . The current PR must be merged after #389 is merged.

File: dapr/proto/runtime/v1/dapr_grpc.pb.go
Comment (ItalyPaleAle) [reply dapr/proto/runtime/v1/dapr_grpc.pb.go:91]: No need for the manually-generated feature unless you also implement the crypto methods.

**Comment by hunter007** [comment]
> 

OK

**Comment by shivamkm07** [comment]
LGTM, but again it has to wait till #389 is merged and then is rebased to master to include the changes

**Comment by hunter007** [comment]
merge code

**Comment by yaron2** [comment]
@hunter007 thanks. the validation examples stage is failing, can you check why?

**Comment by yaron2** [comment]
still failing

**Comment by hunter007** [comment]
> still failing

@yaron2 @artursouza @JoshVanL 
Today I spent lots of time to resolve the question, but until now, it has not resolved.  
I find that it fails when run `./validate.sh actor`.

And I notice that validate-examples haven't  ran after PR 358. I guess something wrong occurred on PR 358, but i'm not sure.

I still resolve it.

**Comment by yaron2** [comment]
@hunter007 we have a regression in the Dapr runtime regarding the handling of reminders data. please revert your last commit that tries to save base64 encoded data.

**Comment by shivamkm07** [comment]
@hunter007 The validate-examples workflow currently is not even testing configuration example. Please add `./validate.sh configuration` here (https://github.com/dapr/go-sdk/blob/d485c1775c37fdf45bbed3e71532888e0c28b857/.github/workflows/validate_examples.yaml#L90) and verify the tests are working fine(the configuration example test, actor test is failing for now due to runtime regression).

**Comment by shivamkm07** [comment]
@hunter007 The configuration validation example is failing in the PR run. This is because the PR is using Dapr 1.10.5 in workflow. Please set `DAPR_REF` variable in the workflow to latest master commit ID in dapr/dapr and then tests(configuration) would run fine

**Comment by shivamkm07** [comment]
> @hunter007 The configuration validation example is failing in the PR run. This is because the PR is using Dapr 1.10.5 in workflow. Please set `DAPR_REF` variable in the workflow to latest master commit of dapr/dapr and then tests(configuration) would run fine

The actor test would fail for now, but that's because the actor regression fix is yet to be merged in master. Once it's done, you would need to update `DAPR_REF` to latest master again. I have validated that actor validation works fine with 1.10.6-rc.2(containing actor patch) cc @mukundansundar @artursouza

**Comment by hunter007** [comment]
> 



> > @hunter007 The configuration validation example is failing in the PR run. This is because the PR is using Dapr 1.10.5 in workflow. Please set `DAPR_REF` variable in the workflow to latest master commit of dapr/dapr and then tests(configuration) would run fine
> 
> The actor test would fail for now, but that's because the actor regression fix is yet to be merged in master. Once it's done, you would need to update `DAPR_REF` to latest master again. I have validated that actor validation works fine with 1.10.6-rc.2(containing actor patch) cc @mukundansundar @artursouza

OK, wait for that actor regression fix has been merged, I'll update the PR.

**Comment by shivamkm07** [comment]
@hunter007 actor regression fix has been merged. Please set `DAPR_REF` in workflow to the latest master commit: https://github.com/dapr/dapr/commits/master and re-run the workflows.

**Comment by shivamkm07** [comment]
@yaron2 Can you please re-run the workflows? And is there a reason these are not getting blocked for approval again?

**Comment by daixiang0** [comment]
LGTM.

**Review by yaron2 — APPROVED** [review]

**Review by yaron2 — APPROVED** [review]

**Pull request merged**
